src/sentry/tasks/post_process.py

```diff
@@ -172,7 +172,7 @@ def index_event_tags(project_id, event_id, tags, group_id=None, **kwargs):
         )
 
         tagvalue, _ = TagValue.objects.get_or_create(
-            project=Project(id=project_id),
+            project=Project.objects.get(id=project_id),
```

Comment:
i still need to figure out how to fix this so we don't actually need to make this query